app/models/dialog/orchestration_template_service_dialog.rb

```diff
@@ -59,7 +62,7 @@ def create_dynamic_dropdown_list(parameter, group, position, dynamic_dropdown, n
       group.dialog_fields.build(
         :type         => "DialogFieldDropDownList",
         :name         => "#{name_prefix}#{parameter.name}",
-        :data_type    => parameter.data_type || "string",
+        :data_type    => "string",
```

Comment:
@bzwei Does this mean we would no longer support dropdown-list with an 'integer' type?  I would image that is something we would still need.

Reply:
`parameter.data_type` is provider specific type. It can be any value or any spelling. We will convert the string back to required value per provider before sending the provisioning request.

Reply:
Since this is only for orchestration and only the drop-downs I am ok with this as long as you are processing the values before sending to the provider.  Previously I was thinking of this issue we ran into (https://bugzilla.redhat.com/show_bug.cgi?id=1478367) where we were passing the value to Azure deployment as an integer instead of a string.